.github/workflows/integration-test.yml

```diff
@@ -5,6 +5,7 @@ on:
   schedule:
```

Comment:
if we are running this on a branch push, including main, then we can probably drop the schedule